test/mock/demo-login-consent-server/templates/login.html

```diff
@@ -35,12 +35,11 @@
 <body onload="setRandomEmail();" class="leading-normal tracking-normal text-white" style="background-color: #f4f1f5;">
 <section class="py-48">
     <p class="text-black text-2xl text-center font-bold">Demo Sign In</p>
-    <div class="container mx-auto flex flex-wrap h-auto lg:w-1/3 w-full">
-        <div class="w-full flex flex-col flex-grow flex-shrink">
-            <div class="flex-none mt-auto rounded-b rounded-t-none overflow-hidden">
-                <div class="flex items-center p-14 justify-center">
-                    <form class="bg-white shadow-xl rounded-xl lg:px-11 sm:px-11 px-4 pt-8
-                    md:flex-wrap md:justify-between w-screen h-auto" action="/login" method="post">
+    <div class="container mx-auto h-auto lg:w-1/3 w-full">
+            <div class="mt-auto rounded-b rounded-t-none overflow-hidden">
+                <div class="items-center p-14 justify-center">
```

Comment:
Is the container not aligned properly horizontally or is it just a part of the screen that was screenshot? I think you're missing `flex` here:
```suggestion
                <div class="flex items-center p-14 justify-center">
```

Reply:
No flex is not required here,  that was just screenshot

Reply:
If it's not required, then you should remove `items-center` and  `justify-center` - these only apply when the container uses flexbox @talwinder50

Reply:
But I'm pretty sure we need it and it used to be there too

Reply:
Yes items-center and justify-center needs to be removed, i have tested it we dont need flex on this div , button dont renders full width if you have flex on all the divs , latest patch works fine

Reply:
Alright then. Could you post a screenshot with the loading state, please?